Breeding-Insight/bi-web
# BI-1764 - DeltaBreed logo

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1764

Updated branding to DeltaBreed

Also implemented:
- BI-1690 - Collapse left hand menu
- Ability to filter for a program in the program selector dropdown
- Hiding the menu on the app home page when not in sandbox mode



# Dependencies
none

# Testing
- Verify that "DeltaBreed" text is used instead of "Breeding Insight Platform"
- Verify that the DeltaBreed logo is used instead of the Breeding Insight logo on the webpage and the favicon

For testing BI-16990
- Verify that on the home page (when not logged in) does not show the menu when not in a sandbox mode
- When logged in, verify that the hamburger icon shows all the time in the top left corner
- When logged in, verify  that clicking the hamburger icon toggles the visibility of the menu
- When logged in, after toggling the menu, verify that the state of the menu persists when navigating between pages


# Checklist:

- [x] I have run TAF: https://github.com/Breeding-Insight/taf/actions/runs/4801946943
